### DISCUSSION TITLE
Ignore App Not Ignoring Filemaker

### DISCUSSION THREAD

**Discussion by peterinjapan** [rxhanson/RectanglePro-Community#588] [General]
I finally got the multiple click window organization setup working nicely using these instructions. One issue I have is, Filemaker wants to use one of the shortcuts I set up, so I tried to activate Ignore App by focusing on Filemaker and selecting Ignore Filemaker Pro in the menu. I saw that it was already active, and unfortunately, all of my keyboard shortcuts are fully active when using Filemaker Pro, even with option toggled on.

Are there any suggestions for how to get Rectangles Pro to ignore keyboard shortcuts when Filemaker Pro is active?

**Comment by rxhanson** [comment]
As a quick sanity check, is ignoring properly working for other apps?

**Comment by peterinjapan** [comment]
Hi, I just activated Ignore Slack for Slack and the keyboard shortcuts are fully active. Here are the keyboard shortcuts I finalized, maybe custom size is somehow not ignoring apps as it should?
